scripts/runAnalysisOverSamples.py

```diff
@@ -288,6 +289,7 @@ def CacheInputs(FileList):
                    if(doCacheInputs and isLocalSample):
                       result = CacheInputs(eventsFile)
                       eventsFile = result[0]
+                      LaunchOnCondor.Jobs_InitCmds.append('if [ -d $TMPDIR ] ; then cd $TMPDIR ; fi;\n')
```

Comment:
Ok I do this.


NB: However I think this will give less unexpected results than not putting it.
First we have one safety net: we erase the files at the end of the job.
Second, it's always better to work in a temporary directory. But this would need to be tested by everyone, so yeah... let's remove it.